### PR TITLE
Versioneer

### PR DESCRIPTION
**Pull request by jan-janssen**
If you run `python setup.py --version` it should show 1.1.2 - the same version as the git tag. At least it works on my fork https://github.com/jan-janssen/h5io .

### CODE REVIEW

**Comment by jan-janssen** [comment]
As there are no git tags on the main repository the version is named: `0.post0.dev80` - based on the 80th commit - in `python setup.py build` on travis.

File: .travis.yml [hunk 1]
```diff
@@ -35,3 +35,19 @@ script:
...
+    on:
+      branch: master
+      condition: "${PYTHON} = 3.6"
+  - provider: pypi
+    user: ${PYPI_USER}
+    password: ${PYPI_TOKEN}
+    allow_failure: true
+    on:
+      tags: true
+      condition: "${PYTHON} = 3.6"
```
[the remark this answers is not in the document]
Comment (larsoner) [reply .travis.yml]: What is your `pypi` username? I can add you as a maintainer there and you can do the config if you're up for it

[the remark this answers is not in the document]
Comment (jan-janssen) [reply .travis.yml]: With the master section you get the prereleases. https://pip.pypa.io/en/stable/reference/pip_install/#install-pre So basically every push to the master is released on pypi and is available via ´pip install --pre h5io´ and the tagged releases are available via ´pip install h5io´.

[the remark this answers is not in the document]
Comment (jan-janssen) [reply .travis.yml]: my pip username is `pyiron´- as that is currently my main project.

**Comment by larsoner** [comment]
> With the master section you get the prereleases. 

I usually just do `pip install https://api.github.com/repos/h5io/h5io/zipball/master` for this. I guess `--pre` is slightly easier

**Comment by jan-janssen** [comment]
@larsoner my suggestion would be to merge this PR now to check if everything is configured correctly.

**Comment by jan-janssen** [comment]
I pushed two releases from my from my branch https://pypi.org/project/h5io/#history so everything seem to be working.

**Comment by larsoner** [comment]
Thanks for working on this @jan-janssen

**Comment by jan-janssen** [comment]
Is it ok if I update the version number to 0.1.1 ?

**Comment by larsoner** [comment]
Yep

**Pull request merged**